src/internal.ts

```diff
@@ -177,3 +177,4 @@ export { revokeIdentityToCreatePortfolios } from '~/api/procedures/revokeIdentit
 export { rotatePrimaryKeyToSecondary } from '~/api/procedures/rotatePrimaryKeyToSecondary';
 export { registerCustomAssetType } from '~/api/procedures/registerCustomAssetType';
 export { bondPolyx } from '~/api/procedures/bondPolyx';
+export { unbondPolyx, Params as UnbondPolyxParams } from '~/api/procedures/unbondPolyx';
```

Comment:
We don't export params out of `src/internal.ts` as those don't get exported for docs. Its better to define the params (`UnbondPolyxParams`) in `src/procedure/types.ts`